certify-service/src/main/java/io/mosip/certify/services/VCIssuanceServiceImpl.java

```diff
@@ -180,7 +180,7 @@ private CredentialResponse<?> getCredentialResponse(String format, VCResult<?> v
     private Optional<CredentialMetadata>  getScopeCredentialMapping(String scope) {
         LinkedHashMap<String, Object> vciMetadata = issuerMetadata.get("latest");
         if(supportedCredentials == null) {
-            supportedCredentials = (LinkedHashMap<String, Object>) vciMetadata.get("credentials_supported");
+            supportedCredentials = (LinkedHashMap<String, Object>) vciMetadata.get("credential_configurations_supported");
         }
 
         Optional<Map.Entry<String, Object>> result = supportedCredentials.entrySet().stream()
```

Comment:
"remove format from the Credential Response" - this change of draft 13 is not taken care. We should also somehow provide backward compatibility